File: redis/lua/update/redis_update.lua
-- ----------------------------------------------------
--	redis_update.lua
--
--				Feb/13/2015
--
-- ----------------------------------------------------
require	('text_manipulate')
require	('redis_manipulate')
-- ----------------------------------------------------
print ("*** 開始 ***")

local key_in=arg[1]
local population_in= 0 + arg[2]

print (key_in,population_in)

local server_host = "host_dbase"
local server_port = 6379

redis_update_proc (server_host,server_port,key_in,population_in)

print ("*** 終了 ***")
-- ----------------------------------------------------

File: redis/fsharp/update/redis_update.fs
// -------------------------------------------------------------------
//
//	redis_update.fs
//
//					Aug/03/2011
//
// -------------------------------------------------------------------
open System
open System.Text
open Microsoft.FSharp.Core
open System.IO

// -------------------------------------------------------------------
[<EntryPoint>]
let main (args : string[] ) =

  printfn "*** 開始 ***"
  let id_in = args.[0]
  let population_in = args.[1]

  printfn "%s" id_in
  printfn "%s" population_in

  let rd = new Redis ("host_dbase",6379)

  redis_manipulate.update_proc (rd,string id_in,int population_in)

  printfn "*** 終了 ***"
  0
// -------------------------------------------------------------------
